curation-migration-datasync/config.ts

```diff
@@ -1,7 +1,7 @@
 const environment = process.env.ENVIRONMENT || 'development';
 const isDev = environment === 'development';
 
-const config = {
+export const config = {
```

Comment:
Sorry about the export changes... I was originally stubbing this (and it's not really possible to stub a default module easily). But then I ran into issues stubbing a different default module (`knex`) so I changed course... but didn't undo it all.